### PR TITLE
Backports 0.18 pr7

### PR DESCRIPTION
**Pull request by Munkybooty**

### CODE REVIEW

File: .travis/test_06_script.sh [hunk 1]
```diff
@@ -0,0 +1,67 @@
+#!/usr/bin/env bash
+#
+# Copyright (c) 2018 The Bitcoin Core developers
+# Distributed under the MIT software license, see the accompanying
+# file COPYING or http://www.opensource.org/licenses/mit-license.php.
+
+export LC_ALL=C.UTF-8
```
Comment (PastaPastaPasta) [inline .travis/test_06_script.sh:7]: 14081 shouldn't just add this file

File: src/noui.cpp [hunk 2]
```diff
@@ -42,12 +42,12 @@ static bool noui_ThreadSafeMessageBox(const std::string& message, const std::str
...
+bool noui_ThreadSafeQuestion(const std::string& /* ignored interactive message */, const std::string& message, const std::string& caption, unsigned int style)
 {
     return noui_ThreadSafeMessageBox(message, caption, style);
 }
 
-static void noui_InitMessage(const std::string& message)
+void noui_InitMessage(const std::string& message)
 {
     LogPrintf("init message: %s\n", message);
 }
```
Comment (PastaPastaPasta) [inline src/noui.cpp:53]: 14162 need to remove line 57

```
// Connect dashd signal handlers
```

File: depends/Makefile [hunk 3]
```diff
@@ -14,7 +14,8 @@ HOST ?= $(BUILD)
 PATCHES_PATH = $(BASEDIR)/patches
 BASEDIR = $(CURDIR)
 HASH_LENGTH:=11
-DOWNLOAD_CONNECT_TIMEOUT:=10
+DOWNLOAD_CONNECT_TIMEOUT:=30
+DOWNLOAD_RETRIES:=3
```
Comment (PastaPastaPasta) [inline depends/Makefile:18]: 14148 shouldn't add retries

File: src/qt/guiutil.cpp [hunk 4]
```diff
@@ -793,52 +793,37 @@ bool SetStartOnSystemStartup(bool fAutoStart)
         CoInitialize(nullptr);
 
         // Get a pointer to the IShellLink interface.
-        IShellLink* psl = nullptr;
+        IShellLinkW* psl = nullptr;
```
Comment (PastaPastaPasta) [inline src/qt/guiutil.cpp:794]: 13734 should drop scoped_array.hpp include

File: src/Makefile.bench.include [hunk 5]
```diff
@@ -65,7 +65,7 @@ if ENABLE_WALLET
 bench_bench_dash_SOURCES += bench/coin_selection.cpp
 endif
 
-bench_bench_dash_LDADD += $(BACKTRACE_LIB) $(BOOST_LIBS) $(BDB_LIBS) $(SSL_LIBS) $(CRYPTO_LIBS) $(MINIUPNPC_LIBS) $(EVENT_PTHREADS_LIBS) $(EVENT_LIBS) $(BLS_LIBS)
+bench_bench_dash_LDADD += $(BACKTRACE_LIB) $(BOOST_LIBS) $(BDB_LIBS) $(CRYPTO_LIBS) $(MINIUPNPC_LIBS) $(EVENT_PTHREADS_LIBS) $(EVENT_LIBS) $(BLS_LIBS)
```
Comment (PastaPastaPasta) [inline src/Makefile.bench.include:68]: drop
```
$(EVENT_PTHREADS_LIBS) $(EVENT_LIBS)
```

File: doc/release-notes/release-notes-0.16.3.md [hunk 6]
```diff
@@ -0,0 +1,88 @@
+Bitcoin Core version 0.16.3 is now available from:
```
Comment (PastaPastaPasta) [inline doc/release-notes/release-notes-0.16.3.md:1]: drop 14251

**Review by PastaPastaPasta — CHANGES_REQUESTED** [review]

File: .travis/test_06_script.sh (hunk 1 above)
Comment (UdjinM6) [reply .travis/test_06_script.sh:7]: yeah, it was created in 13863 originally but we missed it in #4223...

File: src/Makefile.bench.include (hunk 5 above)
Comment (Munkybooty) [reply src/Makefile.bench.include]: So, 14212 keeps destroying my build so i will drop it and try to work on figuring that out

**Comment by Munkybooty** [comment]
@UdjinM6 Do you have any idea why this wont build on windows? I thought 13734 was to blame but it seems not.

**Comment by UdjinM6** [comment]
https://gitlab.com/dashpay/dash/-/jobs/1418036751#L2718 i.e. `fs_tests` failed, so my guess is 13877. The weird thing is I can't reproduce the issue in my repo - https://gitlab.com/UdjinM6/dash/-/pipelines/335701448 is green.

**Review by UdjinM6 — APPROVED** [review]
utACK

**Review by PastaPastaPasta — APPROVED** [review]
utACK

**Pull request merged**